greedy_method/greedy_longest_palindromic_subsequence.py

```diff
@@ -0,0 +1,65 @@
+"""
+The longest palindromic subsequence (LPS) is the longest
+subsequence in a string that is the same when reversed.
+A subsequence is not the same as a substring.
+Unlike substrings, the characters of subsequences are not required
+to occupy consecutive positions.
+
+Explanation:
+https://www.techiedelight.com/longest-palindromic-subsequence-using-dynamic-programming/
```

Comment:
This code is a direct copy from the mentioned link.
@cclauss Please take a look at this.

Reply:
true, only a few names are changed which was recommended by bot